ONSdigital/response-operations-ui
# updated logging messages. updated notify location in config.py

# What has changed
updated logging, when notify notifications don't go out
# How to test?
- spin up ras-rm-docker-dev
- stop docker version of response-ops
- spin up local version of response-ops
- go to localhost:8085 to see local instance of response-ops

in the code:
- comment out the if block (line 27)
- `if not app.config`

In the app:
- "create" a new account
- it should say "Something went wrong"
- check logs in terminal (where you started the local version) to see if the error logging is coming through (lines 48-50)

If you could also see if the logs I've put are sufficient, if more or less information would be required. I have logged what I would find useful when looking at logs but if that may be a concern going forward, please let me know how to improve.
